src/main/java/net/ryanland/empire/bot/command/tutorials/Tutorial.java

```diff
@@ -0,0 +1,39 @@
+package net.ryanland.empire.bot.command.tutorials;
+
+import org.codehaus.groovy.control.CompilationUnit;
```

Comment:
Optimize imports (IntelliJ select the entire project -> Code -> Optimize Imports)